### ISSUE TITLE
"Calendar access denied" but no entry in "Security & Privacy"

### ISSUE THREAD

**Issue by kashnoorani** [sfsam/Itsycal#129]
<img width="427" alt="Screen Shot 2019-05-29 at 8 12 23 AM" src="https://user-images.githubusercontent.com/2106211/58568903-f63bfb80-81e9-11e9-89dc-b445e3e42798.png">
<img width="661" alt="Screen Shot 2019-05-29 at 8 14 24 AM" src="https://user-images.githubusercontent.com/2106211/58568904-f63bfb80-81e9-11e9-92d8-ba9436b99088.png">
<img width="583" alt="Screen Shot 2019-05-29 at 8 16 15 AM" src="https://user-images.githubusercontent.com/2106211/58568967-0d7ae900-81ea-11e9-8c5c-b836fc9a7703.png">

**Comment by kashnoorani** [comment]
Works! Thanks for fixing

Thanks!

**Comment by sfsam** [comment]
Hi. I just realized there is a different problem with the version above. The app auto-updating component doesn't work. I've removed that version from the server and reverted to the previous version. Sorry for the inconvenience.

https://s3.amazonaws.com/itsycal/Itsycal.zip

**Comment by kashnoorani** [comment]
Could you please update this thread when both the issues are fixed? That way I can grab the latest. For the moment, I’d rather keep the version that is working for me.

- K

**Comment by sfsam** [comment]
Should be ok now. 🤞

**Comment by bermudalocket** [comment]
I'm experiencing this same issue on 19C46a. The alternate version linked above unfortunately doesn't work.

**Comment by sfsam** [comment]
@bermudalocket Did it work for you on a previous version of macOS? Are you seeing this issue with the latest version of Itsycal?

**Comment by bermudalocket** [comment]
Yes, I'm on the latest version. Unfortunately I have only just recently discovered Itsycal so I can't comment on previous compatibility.

**Comment by iancbogue** [comment]
I am still having this issue with version 0.12.3 (2024), on macOS Catalina 10.15.4. Nothing shows up in the Calendar section of my Security & Privacy settings, even with the older version linked above (0.11.16).

**Comment by bermudalocket** [comment]
I'm pretty sure this is a wide-ranging issue in macOS 10.15+ that's caused by disabling SIP. The problem is resolved by re-enabling SIP and then launching the app. SIP can be disabled again after permissions are granted.

**Comment by pauloewerton** [comment]
I'm having this issue too, I reported by e-mail but thought of leaving a comment here as well. Currently installed version of Itsycal: 0.13.4 (also tried with latest available version), running on macOS Catalina 10.15.7. I tried out the SIP disabling tip, but no success.

**Comment by 0jay** [comment]
> I'm having this issue too, I reported by e-mail but thought of leaving a comment here as well. Currently installed version of Itsycal: 0.13.4 (also tried with latest available version), running on macOS Catalina 10.15.7. I tried out the SIP disabling tip, but no success.

Same problem, also Catalina
